nncf/automl/environment/quantization_env.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, qmodel: NNCFNetwork, per_quantizer_config_space: Dict[Quantiz
         self._nparam_map = OrderedDict()
         for qid, qconfig_space in per_quantizer_config_space.items():
             if isinstance(qid, WeightQuantizerId):
-                self._bw_space_map[qid] = [qconf.bits for qconf in qconfig_space]
+                self._bw_space_map[qid] = [qconf.num_bits for qconf in qconfig_space]
```

Comment:
probably `bitwidth` is better to be aligned with the rest of code.

Reply:
I went for quantizer config attribute name and quantizer module attribute name alignment instead.

Reply:
while you are updating checkpoints it's a good moment to align quantizer module attribute as well. 
It would be great using `bitwidth` everywhere, like in documentation and precision init algos.

Reply:
@alexsu52 if I change `num_bits` to `bitwidth` globally, will TF checkpoints be affected? Do they allow for easy conversion w.r.t. this renaming?

Reply:
Also, as I've discovered just now, `tensorflow.python.ops.gen_array_ops.fake_quant_with_min_max_vars_per_channel` uses `num_bits` as a corresponding keyword argument. Wouldn't it be better to be aligned with this as well?

Reply:
@AlexKoff88, I remember you proposed `bitwidth` name. What do you think about changing it to `num_bits`?

Reply:
Both are meaningful. I don't mind.

Reply:
> @alexsu52 if I change num_bits to bitwidth globally, will TF checkpoints be affected? Do they allow for easy conversion w.r.t. this renaming?

TF checkpoints will not be affected. Thus I don't see any concern for renaming 

@vshampor @ljaljushkin I would like us to use consistent naming of key parameters in various algorithms and their implementations.  BTW, TF uses `num_bits`.

Reply:
> Also, as I've discovered just now, `tensorflow.python.ops.gen_array_ops.fake_quant_with_min_max_vars_per_channel` uses `num_bits` as a corresponding keyword argument. Wouldn't it be better to be aligned with this as well?

ok, I will rename `bitwidth` to `num_bits` in a separate PR then